### PR TITLE
Fixes part of #4177: Dark mode Topic Activity Part 2 (Info Tab)

### PR DESCRIPTION
**Pull request by MohitGupta121**
## Explanation

Fixes part of #4177 : Dark mode Topic Activity Part 2 (Info Tab)

Dark Mode Mockup:-
https://xd.adobe.com/view/c05e9343-60f6-4c11-84ac-c756b75b940f-950d/screen/3b067181-e5e0-4a5b-bbb8-ab5eda952549/specs/

|Default|Dark Mode|
|:----------------------------------------:|:-----------------------------------------:|
|  | Screenshot_2022-05-31-12-23-27-332_com android camera |

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Comment by MohitGupta121** [comment]
@rt4914 don't know why these test are failing. I only resolve confilct and update this branch as per Develop. It's buiding succesfull. Also In Test Fail it not specify any particular error.

File: app/src/main/res/drawable/thumbnail_gradient.xml [hunk 1]
```diff
@@ -4,6 +4,6 @@
   <gradient
     android:angle="270"
     android:startColor="#00FFFFFF"
```
Comment (rt4914) [inline app/src/main/res/drawable/thumbnail_gradient.xml]: Please create a similar value for startColor too

File: app/src/main/res/values-night/color_palette.xml [hunk 2]
```diff
@@ -90,4 +90,6 @@
   <color name="color_palette_confetti_red_color">@color/color_def_confetti_rosy_pink</color>
   <color name="color_palette_confetti_yellow_color">@color/color_def_confetti_bright_gold</color>
   <color name="color_palette_confetti_blue_color">@color/color_def_confetti_crystal_blue</color>
+  <color name="color_palette_forward_arrow_button_color">@color/color_def_oppia_turquoise</color>
```
Comment (rt4914) [inline app/src/main/res/values-night/color_palette.xml:93]: Please create this color in `values/color_palette.xml` too.

File: app/src/main/res/values/component_colors.xml [hunk 3]
```diff
@@ -246,5 +246,14 @@
   <color name="component_color_confetti_red_color">@color/color_palette_confetti_red_color</color>
   <color name="component_color_confetti_yellow_color">@color/color_palette_confetti_yellow_color</color>
   <color name="component_color_confetti_blue_color">@color/color_palette_confetti_blue_color</color>
+  <!-- Topic Info Activity -->
```
Comment (rt4914) [inline app/src/main/res/values/component_colors.xml]: Change this to `Topic Info Fragment`

**Review by rt4914 — CHANGES_REQUESTED** [review]
@MohitGupta121 Suggested changes

File: app/src/main/res/drawable/thumbnail_gradient.xml (hunk 1 above)
Comment (MohitGupta121) [reply app/src/main/res/drawable/thumbnail_gradient.xml]: > Please create a similar value for startColor too

Actually startColor not changing in dark_mode it's same. Still I have to make value?

File: app/src/main/res/values-night/color_palette.xml (hunk 2 above)
Comment (MohitGupta121) [reply app/src/main/res/values-night/color_palette.xml:93]: > 

Okay sure, actually it's already defined.

File: app/src/main/res/values/component_colors.xml (hunk 3 above)
Comment (MohitGupta121) [reply app/src/main/res/values/component_colors.xml]: > Change this to `Topic Info Fragment`

okay

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks.

**Pull request merged**